src/style/home.scss

```diff
@@ -0,0 +1,46 @@
+@import 'constants.scss';
```

Comment:
In react its usually more common to put the `scss` files next to the corresponding `tsx` files (as they are really all part of the same thing).

Could you move this file?